java/serve/src/main/java/io/ray/serve/deployment/DeploymentCreator.java

```diff
@@ -250,7 +250,8 @@ public DeploymentLanguage getDeploymentLanguage() {
     return deploymentLanguage;
   }
 
-  public void setDeploymentLanguage(DeploymentLanguage deploymentLanguage) {
+  public DeploymentCreator setDeploymentLanguage(DeploymentLanguage deploymentLanguage) {
```

Comment:
It's not related to this PR, but I really think it's better to change this method name to `setLanguage()` because this class is `DeploymentCreator`.

Reply:
Resolved. rename to `setLanguage()`